app/services/ccr/expenses_adapter.rb

```diff
@@ -0,0 +1,109 @@
+module CCR
+  class ExpensesAdapter
+    attr_reader :expense
+
+    TRANSLATION_TABLE = {
```

Comment:
is there anyway we could refactor this to remove the repetition of the string values?

Reply:
...although the repetition is clear 🤔